controls/pcidss_ocp4.yml

```diff
@@ -1838,137 +1838,153 @@ controls:
   rules: []
 
 - id: Req-10.1
-  title: 10.1 Implement audit trails to link all access to system components to each
-    individual user.
+  title: >-
+    10.1 Implement audit trails to link all access to system components to each individual
+    user.
+  description: |-
+    All actions taken by users of OpenShift are logged and capable of being used to satisfy audit requirements.
```

Comment:
I've been doing these assessments on the `notes` section. What do you think about moving these to that key?

Reply:
Sounds good, I'll fix it